packages/material-ui-lab/src/TimelineItem/TimelineItem.js

```diff
@@ -58,19 +86,23 @@ const TimelineItem = React.forwardRef(function TimelineItem(props, ref) {
     }
   });
 
+  const styleProps = {
+    ...props,
+    align,
+    hasOppositeContent,
+  };
+
+  const classes = useUtilityClasses(styleProps);
+
   return (
     <TimelineItemContext.Provider
-      value={{ classes: { content: classes.content, oppositeContent: classes.oppositeContent } }}
+      value={{
+        classes: { content: classes.content, oppositeContent: classes.oppositeContent },
```

Comment:
I don't think we need this anymore. As far as I remember, we needed this in the past because we could not rely on the global classes. You can now safely use the classes for the `MuiTimelineContent-root` and `MuiTimelineOppositeContent-root`. Also you don't need to read this from content anymore in these two components.